### PR TITLE
Is adjustments

### PR DESCRIPTION
**Pull request by FranFin**

### CODE REVIEW

File: docs/ros1.rst [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+If you want to test it the other way around, launch the *ROS1* :code:`echo` with the command:
+
+.. code-block:: bash
+
+    source /opt/ros/melodic/setup.bash
+    rostopic echo /chatter
+
+Then launch the HelloWorld as a :code:`publisher`, and *eProsima Integration-Service* using the :code:`soss` command
+followed by the `dds_ros1.yaml <https://github.com/eProsima/SOSS-DDS/blob/doc/examples/examples/ros1/dds_rps1.yaml>`__
```
Comment (LuisGP) [inline docs/ros1.rst:127]:
```suggestion
followed by the `dds_ros1.yaml <https://github.com/eProsima/SOSS-DDS/blob/doc/examples/examples/ros1/dds_ros1.yaml>`__
```

File: docs/fiware_ros1.rst [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  And then build the packages by running:
+
+  .. code-block:: bash
+
+      colcon build
+
+Before proceeding, note that the
+`dds_fiware_ros1.yaml <https://github.com/eProsima/SOSS-DDS/tree/doc/examples/examples/fiware/dds_fiware_ros1.yaml>`__
+and the
+`fiware_dds_ros1.yaml <https://github.com/eProsima/SOSS-DDS/tree/doc/examples/examples/fiware/fiware_dds_sros1.yaml>`__
```
Comment (LuisGP) [inline docs/fiware_ros1.rst:105]:
```suggestion
`fiware_dds_ros1.yaml <https://github.com/eProsima/SOSS-DDS/tree/doc/examples/examples/fiware/fiware_dds_ros1.yaml>`__
```

File: docs/usecases_content.rst [hunk 3]
```diff
@@ -40,18 +40,17 @@ in the introductory section :ref:`Getting Started <getting started>`.
 Please make sure to follow all the steps described in the document before proceeding.
 
 Whenever you run the :code:`colcon build` command in the examples provided, if any package is missing dependencies
-**causing the compilation to fail**, you can add the flag :code:`--packages-up-to soss-dds-test soss-ros2-test`
-as follows:
+**causing the compilation to fail**, you can add the flag :code:`--packages-up-to soss-dds-test as follows:
```
Comment (LuisGP) [inline docs/usecases_content.rst:43]:
```suggestion
**causing the compilation to fail**, you can add the flag :code:`--packages-up-to soss-dds-test` as follows:
```

**Comment by richiware** [comment]
Buildstatus:[BuildStatus](http://jenkins.eprosima.com:8080/job/integration_service_docs_manual/2/)

**Comment by richiware** [comment]
Buildstatus:[BuildStatus](http://jenkins.eprosima.com:8080/job/integration_service_docs_manual/4/)

**Comment by richiware** [comment]
Buildstatus:[BuildStatus](http://jenkins.eprosima.com:8080/job/integration_service_docs_manual/5/)

**Comment by richiware** [comment]
Buildstatus:[BuildStatus](http://jenkins.eprosima.com:8080/job/integration_service_docs_manual/6/)

**Comment by richiware** [comment]
Buildstatus:[BuildStatus](http://jenkins.eprosima.com:8080/job/integration_service_docs_manual/7/)

**Pull request merged**